### DISCUSSION TITLE
Custom authorizer connect doesn't work by enable sign

### DISCUSSION THREAD

**Discussion by arlyxiao** [aws/aws-iot-device-sdk-js-v2#451] [Q&A] [answered]
Anyone can run the demo by enable sign authorizer?
https://github.com/aws/aws-iot-device-sdk-js-v2/tree/main/samples/browser/custom_authorizer_connect

**Comment by jmklix** [comment, marked as answer]
You should be able to run the demo by following the documentation. Is there a specific part that you're stuck on/having trouble with?
